### PR TITLE
Update to v0.6.2 release

### PR DESCRIPTION
**Pull request by tonyhoo**
*Issue #, if available:*

*Description of changes:*
Update versions for 0.6.2 release with release notes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by gradientsky — APPROVED** [review]

File: docs/whats_new/v0.6.2.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Version 0.6.2
+
+v0.6.2 is a security and bug fix release.
+
+This release is backward compatible to 0.6.0 and 0.6.1. As always, only load previously trained models using the same version of AutoGluon that they were originally trained on.
```
Comment (Innixma) [inline docs/whats_new/v0.6.2.md:5]: "This release is backward compatible to 0.6.0 and 0.6.1.":

I never tested that, nor do I suspect did anyone else in any module test that. I suspect it doesn't hold True in all cases. I don't think we want to claim backwards compatibility.

Comment (Innixma) [reply docs/whats_new/v0.6.2.md:5]: Or do you mean API is same? In that case yes. But you shouldn't be loading a v0.6.1 predictor in v0.6.2, we didn't test that.

File: docs/whats_new/v0.6.2.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Fix missing Predictor API doc (#2573) - @gidler
+- 2023 Roadmap Update (#2590) - @lnnixma
+- Image classifiction tutorial update for bytearray (#2598) - @suzhoum
+- Fix broken tutorial index links (#2617) - @shchur
+- Improve timeseries quickstart tutorial (#2653) - @shchur
+
+
+## Bug Fixes / Security
+
+- [multimodal] Refactoring and bug fixes(#2554, #2541, #2477, #2569, #2578, #2613, #2620, #2630, #2633, #2635, #2647, #2645, #2652, #2659) - @zhiqiangdon, @yongxinw, @FANGAreNotGnu, @sxjscience, @lnnixma
```
Comment (Innixma) [inline docs/whats_new/v0.6.2.md:26]: @lnnixma -> @innixma

I not L (I wanted to change the username to all lowercase, but for whatever reason GitHub allows you to change your alias to something else, but not to the same name with different casing...)

File: docs/whats_new/v0.6.2.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Version 0.6.2
+
+v0.6.2 is a security and bug fix release.
+
+This release is backward compatible to 0.6.0 and 0.6.1. As always, only load previously trained models using the same version of AutoGluon that they were originally trained on.
+Loading models trained in different versions of AutoGluon is not supported.
+
+See the full commit change-log here: https://github.com/autogluon/autogluon/compare/v0.6.1...v0.6.2
+
+This version supports Python versions 3.7 to 3.9. 0.6.x are the last releases that will support Python 3.7.
```
Comment (Innixma) [inline docs/whats_new/v0.6.2.md:10]: Add special thanks to first time contributors

File: docs/whats_new/v0.6.2.md (hunk 2 above)
Comment (tonyhoo) [reply docs/whats_new/v0.6.2.md]: Didn't notice. Fixed

File: docs/whats_new/v0.6.2.md (hunk 1 above)
Comment (tonyhoo) [reply docs/whats_new/v0.6.2.md]: Will remove that statement. Got similar statement from priror 0.4.3

File: docs/whats_new/v0.6.2.md (hunk 3 above)
Comment (tonyhoo) [reply docs/whats_new/v0.6.2.md:10]: Added @daikikatsuragawa and @yzhliu ti special thanks

**Review by liangfu — APPROVED** [review]

**Pull request merged**